app/views/games/show.html.erb

```diff
@@ -1 +1,10 @@
-<h1>Show page!</h1>
\ No newline at end of file
+<h3 class='col-12 offset-4'><%= @game.name %></h3>
+<br>
+<div class="booyah-box col-10 offset-4">  
+  <div class="chessboard">

```

Comment:
@saj528 it would be great to prepare the board to generate 64 div's - so that each cell is later easily accessible. It will make it easy for us to place the pieces on the board later on...